static/js/stats/stats.js

```diff
@@ -55,6 +56,26 @@ function format_date(date, include_hour) {
     return month_str + ' ' + day + ', ' + year;
 }
 
+function format_date_for_last_update(last_update) {
+    last_update = new Date(last_update * 1000);
+    var locale_date = last_update.toLocaleDateString();
```

Comment:
I should have pointed at a different resource, e.g. see the bottom of https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString for how to get a nicer format.